docs/using-the-compiler.rst

```diff
@@ -639,4 +639,4 @@ Error Types
 12. ``FatalError``: Fatal error not processed correctly - this should be reported as an issue.
 13. ``YulException``: Error during Yul code generation - this should be reported as an issue.
 14. ``Warning``: A warning, which didn't stop the compilation, but should be addressed if possible.
-15. ``Info``: Information that the compiler thinks the user might find useful, but is not dangerous and does not necessarily need to be addressed.
\ No newline at end of file
+15. ``Info``: Information that the compiler thinks the user might find useful, but is not dangerous and does not necessarily need to be addressed.
```

Comment:
This is a newline that was inserted (automatically by my text editor) at the end of the file. I guess no harm in leaving it?

Reply:
Nope, in fact most of our docs have the newline at the end. All good.